assets/scss/layouts/products/_productSwatch.scss

```diff
@@ -45,6 +44,13 @@ $second_value : str-slice($value_of_swatch_size, $position_of_x + 1);
         transform: translate(-50%, 0);
         visibility: visible;
         z-index: 5000;
+        padding: 3px;
+        border: 1px solid stencilColor("input-border-color-active");
+
+    }
+    .form-option-image {
```

Comment:
Why are we changing swatch styles in context of this task?

Reply:
It allows to remove overflow that cause truncation of wishlist